### PR TITLE
Volunteer routes

### PR DESCRIPTION
**Pull request by Artsyy**
I didn't know wether I could use the Hacker validators and handlers but assumed we could.

### CODE REVIEW

File: HackMerced/2017F/src/server/src/routes/volunteer.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      handler: hackerHandlers.postSubmit,
+      validate: hackerValidators.postSubmit,
+      auth: {
+          strategy: 'token',
+      }
+    }
+};
+
+// [POST] /me
+export const postMe = {
```
Comment (4shub) [inline HackMerced/2017F/src/server/src/routes/volunteer.js]: remove?

File: HackMerced/2017F/src/server/src/routes/index.js [hunk 2]
```diff
@@ -1,10 +1,15 @@
 import hackerRoutes from './hacker';
 import homeRoutes from './home';
+import volunteerRoutes from './volunteer';
 
 let routes = [
   ...hackerRoutes,
 ];
 
+let routes = [
```
Comment (zzmarkzz321) [inline HackMerced/2017F/src/server/src/routes/index.js]: Remove duplicate variable and move inside of one

File: HackMerced/2017F/src/server/src/routes/volunteer.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import Joi from 'joi';
+import Boom from 'boom';
+import { volunteerHandlers } from '../handlers'
+import { volunteerValidators } from '../validators'
+
+// [POST] /login
+export const postVolunteer = {
```
Comment (zzmarkzz321) [inline HackMerced/2017F/src/server/src/routes/volunteer.js:7]: We already have a route for ``` /signup ```, create a new unique route for volunteers. 
ex:
```plaintext
/volunteers/signup
```

File: HackMerced/2017F/src/server/src/handlers/volunteer.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    return boomMessage;
+  } else {
+    return Boom.badImplementation(err.message);
+  }
+}
+
+export const volunteerHandlers = {
+  postVolunteer: (req, reply) => {
+    axios
+      .post(TOMOE_URI + '/volunteers', req.payload)
```
Comment (zzmarkzz321) [inline HackMerced/2017F/src/server/src/handlers/volunteer.js]: ``` /volunteers ``` is not an endpoint. We'll most likely be using hackers and modifying the survey field.

**Review by zzmarkzz321 — CHANGES_REQUESTED** [review]

File: HackMerced/2017F/src/server/src/validators/volunteer.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+import { notMercedOptions } from '../../../app/src/constants'
+
+export const volunteerValidators = {
+	postVolunteer:{
+		//tomoe
+	},
+	getMe:{
+		payload:{
+	    	id: Joi.string().required()
```
Comment (zzmarkzz321) [inline HackMerced/2017F/src/server/src/validators/volunteer.js]: indent and close payload json

**Review by zzmarkzz321 — CHANGES_REQUESTED** [review]
Also ``` export * from ./volunteer ``` inside of ``` index.js``` for validators and handlers

**Comment by zzmarkzz321** [comment]
Looks good; Waiting on me to finish the rest of the back end. Once my PR has been made and I test the entire feature, we can merge both as one.

**Review by zzmarkzz321 — APPROVED** [review]

**Pull request merged**